src/SFA.DAS.Commitments.Support.SubSite/App_Start/FilterConfig.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Web;
+using System.Web.Mvc;
+
+namespace SFA.DAS.Commitments.Support.SubSite
+{
+    public class FilterConfig
```

Comment:
This class contains only a static method and is not meant to be instantiated, so can be marked static.